addons/picklezip-mm/README.md

```diff
@@ -1,4 +1,5 @@
-# pickleZip-mm (pzMM) v1.0
+# picklezip-mm (pzmm) v1.0 - Obsolete
+_**Note:** The pzmm package has been merged with the [python-sasctl](https://github.com/sassoftware/python-sasctl) package, and is now the [PZMM module](https://github.com/sassoftware/python-sasctl/tree/master/src/sasctl/pzmm)._
 
```

Comment:
As above, let's drop the capitalization of pzmm.

Reply:
You use "PZMM" in the sasctl/pzmm README.md.  I thought that is what we decided on. Are you going to fix all of the references in the python-sasctl GitHub repository to be pzmm instead of PZMM?
https://github.com/sassoftware/python-sasctl/tree/master/src/sasctl/pzmm